### PR TITLE
ecommerce theme role

### PR DESCRIPTION
**Pull request by amirtds**
Configuration Pull Request
---

Make sure that the following steps are done before merging

  - [ ] @devops team member has commented with :+1:
  - [ ] are you adding any new default values that need to be overridden when this goes live?  
    - [ ] Open a ticket (DEVOPS) to make sure that they have been added to secure vars.
    - [ ] Add an entry to the CHANGELOG.

### CODE REVIEW

**Comment by amirtds** [comment]
[(6) UW: come up with plan for modifying ecommerce templates](https://trello.com/c/S9eR3JqP/2033-6-uw-come-up-with-plan-for-modifying-ecommerce-templates)

File: playbooks/roles/ecommerce_theme/tasks/main.yml [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+# create themes directory in /edx/var/ecommerce if it doesn't exist
+- name: Create themes directory
+  file:
+    path: "/edx/var/ecommerce/themes"
+    owner: ecommerce
+    group: www-data
+    recurse: yes
+    mode: 0755
+    state: directory
+
```
Comment (mrgnr) [inline playbooks/roles/ecommerce_theme/tasks/main.yml]: Remove blank line.

File: playbooks/roles/ecommerce_theme/tasks/main.yml [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+# set right permissions on the site theme
+- name: Create themes directory
+  file:
+    path: "/edx/var/ecommerce/themes/{{ ECOMMERCE_DEFAULT_SITE_THEME }}"
+    owner: ecommerce
+    group: www-data
+    recurse: yes
+    mode: 0755
+    state: directory
+
```
Comment (mrgnr) [inline playbooks/roles/ecommerce_theme/tasks/main.yml]: Remove blank line.

File: playbooks/roles/ecommerce_theme/tasks/main.yml [hunk 3]
```diff
@@ -0,0 +1,82 @@
+####### git update didn't work properly till i update my ansible to 2.2.1
+
+# create themes directory in /edx/var/ecommerce if it doesn't exist
+- name: Create themes directory
+  file:
+    path: "/edx/var/ecommerce/themes"
+    owner: ecommerce
```
Comment (mrgnr) [inline playbooks/roles/ecommerce_theme/tasks/main.yml]: Change to `owner: "{{ ecommerce_user }}"`.

File: playbooks/roles/ecommerce_theme/tasks/main.yml [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+    version: "{{ ECOMMERCE_THEME_BRANCH }}"
+    accept_hostkey: yes
+  tags:
+    - ecommerce_theme
+
+# set right permissions on the site theme
+- name: Create themes directory
+  file:
+    path: "/edx/var/ecommerce/themes/{{ ECOMMERCE_DEFAULT_SITE_THEME }}"
+    owner: ecommerce
```
Comment (mrgnr) [inline playbooks/roles/ecommerce_theme/tasks/main.yml]: Change to `owner: "{{ ecommerce_user }}"`.

**Comment by tkeemon** [comment]
@amirtds two changes: 

- [ ] Make your change to `ficus_pro.yml` in the other Ficus playbooks. 
- [ ] The `ecommerce` role isn't being run in any of the ficus playbooks. Let's add that role to each playbook behind a feature flag, similarly to how you added `ecommerce_theme` to `ficus_pro.yml`.

**Pull request merged**